spec/features/home_spec.rb

```diff
@@ -45,11 +47,26 @@
     it "should redirect to main page" do
       user_one = create(:user, email: "john@example.com", password: "john", password_confirmation: "john")
       user_two = create(:user, email: "mike@example.com", password: "mike", password_confirmation: "mike")
-      create(:card, original_text: "Test", translated_text: "Тест", user: user_one)
-      create(:card, original_text: "Test", translated_text: "Тест", user: user_two)
+      deck_one = create(:deck, name: "Johndeck", current: false, user: user_one)
+      deck_two = create(:deck, name: "Mikedeck", current: false, user: user_two)
+      create(:card, original_text: "Test", translated_text: "Тест", user: user_one, deck: deck_one)
+      create(:card, original_text: "Test", translated_text: "Тест", user: user_two, deck: deck_two)
       login_user_post(user_one.email, "john")
       visit edit_card_path(user_two.cards.first)
       expect(current_path).to eq(root_path)
     end
   end
+
+    context "when there is a current deck" do
+    it "should show card from the current deck" do
+      user = create(:user, email: "john@example.com", password: "john", password_confirmation: "john")
+      current_deck = create(:deck, name: "Currentdeck", current: true, user: user)
+      deck = create(:deck, name: "Deck", current: false, user: user)
+      create(:card, original_text: "Test current", translated_text: "Тест текущий", user: user, deck: current_deck)
+      create(:card, original_text: "Test", translated_text: "Тест", user: user, deck: deck)
+      login_user_post(user.email, "john")
+      visit root_url
+      expect(page).to have_content("Test current")
+    end
+  end
```

Comment:
`end` at 71, 2 is not aligned with `context "when there is a current deck" do` at 60, 4